dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/SSOCookieHelperImpl.java

```diff
@@ -544,6 +547,27 @@ public void addSSOCookiesToResponse(Subject subject, HttpServletRequest req, Htt
             return;
         }
 
+        SubjectHelper subjectHelper = new SubjectHelper();
+
+        if (subjectHelper.getHashtableFromSubject(subject, disableSsoLtpaCookie) == null) {
```

Comment:
There may be a defect in this path if someone adds AuthenticationConstants.INTERNAL_DISABLE_SSO_LTPA_COOKIE with a value of false to the hashtable. The cookie should still be created in that case.

Reply:
It's an internal property and default is false. I check for true any way